extensions/amp-consent/amp-consent.md

```diff
@@ -434,37 +551,35 @@ Example:
 
 Use the `<amp-consent>` component which allows configuring a custom UI with `Accept`, `Reject` and `Dismiss` states. It is up to each amp-component vendor how they interpret these states, so please read your vendor's documentation carefully.
 
-Note that you must configure either the [`promptIfUnknownForGeoGroup`](#promptifunknownforgeogroup) attribute or the checkConsentHref request.
+To collect consent from all users, you put `consentRequired: true` in the config of `<amp-consent>`.
```

Comment:
We still need to declare the old message. Also document if `<amp-consent>` can't find the `consentRequired` value, it will default to `false`. (Not ideal, but that's the current behavior)

And I don't think this will be the case. No need to address such a corner use case here.